### PR TITLE
Learn: Plonky page

### PR DESCRIPTION
**Pull request by kmurphypolygon**
Plus a few other style alignment changes.

### CODE REVIEW

File: docs/learn/plonky.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+comments: true
+---
+
+Plonky relates to active research and the development of libraries containing performant cryptographic functions for use in zero-knowledge proof systems.
```
Comment (EmpieichO) [inline docs/learn/plonky.md]: It's not clear what point this sentence is driving home ... especially as it's the first sentence of the doc. 
If it is about the name "Plonky", I would say the subsequent sentence do a better job. So please revise or delete.

File: docs/learn/plonky.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+[Plonky](https://github.com/0xPolygonZero/plonky?tab=readme-ov-file) was Polygon's original implementation of a zk-SNARK computational cryptographic library based on [Plonk](https://eprint.iacr.org/2019/953) with some customizations. The original library was decommissioned in 2021 to be replaced by Plonky2.
+
+## Plonky 2
+
+Plonky2 is a performant Rust library of cryptographic functions that includes a SNARK implementation based on techniques from [Plonk](https://eprint.iacr.org/2019/953) and [FRI](https://drops.dagstuhl.de/storage/00lipics/lipics-vol107-icalp2018/LIPIcs.ICALP.2018.14/LIPIcs.ICALP.2018.14.pdf) as well as tools such as Starky.
```
Comment (EmpieichO) [inline docs/learn/plonky.md:11]: For an interested reader, it would be beneficial to provide a link to "Starky"

File: docs/learn/plonky.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+!!! tip "More info"
+    - Check out the [Plonky2 tutorial](https://polymerlabs.medium.com/a-tutorial-on-writing-zk-proofs-with-plonky2-part-i-be5812f6b798) for more information on the library and how to use it.
+    - Check out the [Plonky2 whitepaper](https://github.com/0xPolygonZero/plonky2/blob/main/plonky2/plonky2.pdf).
+
+## Plonky 3
+
+Plonky3 is an ongoing effort to provide a new and enhanced cryptographic library which aims to further improve on the speed and efficiency of recursive zero-knowledge proofs. It contains optimizations for newer CPU specifications.
+
+It implements polynomial IOPs, such as PLONK and STARKS, and commitment schemes such as Brakedown. Check out the [Plonky3 README](https://github.com/Plonky3/Plonky3) for an update on what is included.
```
Comment (EmpieichO) [inline docs/learn/plonky.md]: A link to the Brakedown paper would be nice. 
I found this paper, "Brakedown" … https://eprint.iacr.org/2021/1043.pdf  
However, it seems like it’s a SNARK … so not sure about the Commitment scheme part ?

**Review by EmpieichO — APPROVED** [review]

File: docs/learn/plonky.md (hunk 1 above)
Comment (kmurphypolygon) [reply docs/learn/plonky.md]: updated

File: docs/learn/plonky.md (hunk 2 above)
Comment (kmurphypolygon) [reply docs/learn/plonky.md:11]: I did search for one but couldn't find anything specific... pinging you now for help ...

**Pull request merged**